File: ChatCI/src/entities/Usuario.java
public class Usuario {
    private String nome;
    private String email;
    private String tipo; // Aluno, Professor, Administrador
    private boolean bloqueado;

    public Usuario(String nome, String email, String tipo) {
        this.nome = nome;
        this.email = email;
        this.tipo = tipo;
        this.bloqueado = false;
    }

    public void bloquear() {
        this.bloqueado = true;
    }

    public void desbloquear() {
        this.bloqueado = false;
    }

    public String getNome() {
        return nome;
    }

    public String getEmail() {
        return email;
    }

    public String getTipo() {
        return tipo;
    }

    public boolean isBloqueado() {
        return bloqueado;
    }
}

File: ChatCI/entities/usuario.py
class Usuario:
   def __init__(self, nome, email, tipo):
      self.nome = nome
      self.email = email
      self.tipo = tipo  # Aluno, Professor, Administrador
      self.bloqueado = False

   def bloquear(self):
      self.bloqueado = True

   def desbloquear(self):
      self.bloqueado = False
